### ISSUE TITLE
Test suite: ambiguous module for doctest when running with cabal

### ISSUE THREAD

**Issue by snoyberg** [peti/language-nix#1]
```
Test suite spec: RUNNING...

src/Language/Nix/Binding.hs:8:8:
    Ambiguous module name ‘Text.PrettyPrint.HughesPJClass’:
      it was found in multiple packages:
      pretty-1.1.2.0@prett_7UQTOB05U7lIYPkFOVraeR pretty-class-1.0.1.1@prett_AZgjvpRF5qfE7LskfU96E3 prettyclass-1.0.0.0@prett_7UaeeUCHbpBCOrvJr5BDLz
Test suite spec: FAIL
Test suite logged to: /home/stackage/work/logs/nightly/language-nix-2/test-run.out
```

**Comment by peti** [comment]
Duh, trying to substitute recent versions of `pretty` with `prettyclass` on old compilers has been one certifiable nightmare. I'll give up on the `Pretty` class and use `Text` instead. :disappointed:

**Comment by peti** [comment]
Fixed in [version 2.1](http://hackage.haskell.org/package/language-nix-2.1).
